src/Week1/HW01Tests.hs

```diff
@@ -0,0 +1,70 @@
+-- CIS 194, Spring 2015
+--
+-- Test cases for HW 01
+
+module Week1.HW01Tests where
+
+import Week1.HW01
+import Week1.Testing
+
+-- Exercise 1 -----------------------------------------
+
+testLastDigit :: (Integer, Integer) -> Bool
+testLastDigit (n, d) = lastDigit n == d
+
+testDropLastDigit :: (Integer, Integer) -> Bool
+testDropLastDigit (n, d) = dropLastDigit n == d
+
+ex1Tests :: [Test]
+ex1Tests = [ Test "lastDigit test" testLastDigit
+             [(123, 3), (1234, 4), (5, 5), (10, 0), (0, 0)]
+           , Test "dropLastDigit test" testDropLastDigit
+             [(123, 12), (1234, 123), (5, 0), (10, 1), (0,0)]
+           ]
+
+-- Exercise 2 -----------------------------------------
+testtoRevDigits :: (Integer, [Integer]) -> Bool
+testtoRevDigits (n, d) = toRevDigits n == d
+ex2Tests :: [Test]
+ex2Tests = [Test "toRevDigits" testtoRevDigits
+             [(341, [1,4,3]), (8341, [1,4,3,8]), (0, [])]]
```

Comment:
Save some typing by reusing functions testF1, testF2 etc from the Testing module

```suggestion
ex2Tests :: [Test]
ex2Tests = [testF1 "toRevDigits" toRevDigits
             [(341, [1,4,3]), (8341, [1,4,3,8]), (0, [])]]
```

Similarly for other tests below..